cni/README.md

```diff
@@ -1,15 +1,17 @@
 ## `cni` ##
 
-There are a wide variety of different [CNI][cni] network configurations. This
+There are a wide variety of different [CNI](https://github.com/containernetworking/cni) network configurations. This
 directory just contains an example configuration that can be used as the
 basis for your own configuration.
 
 To use this configuration, place it in `/etc/cni/net.d` (or the directory
 specified by `cni_config_dir` in your `libpod.conf`).
 
-In addition, you need to install the [CNI plugins][cni] necessary into
-`/opt/cni/bin` (or the directory specified by `cni_plugin_dir`). The
-two plugins necessary for the example CNI configurations are `portmap` and
-`bridge`.
+For example a basic network configuration can be achieved with:
 
-[cni]: https://github.com/containernetworking/plugins
+```bash
+sudo mkdir -p /etc/cni/net.d
+curl -qsSL https://raw.githubusercontent.com/containers/libpod/master/cni/87-podman-bridge.conflist | sudo tee /etc/cni/net.d/99-loopback.conf
```

Comment:
imho opinion the 87 conf should be called the 87 conf.  loopback is assumed by cni

Reply:
LIttle confused by this.   I think you're asking to make the target here 87-podman-bridge.conf instead of 99-loopback.conf.  I'll make that change, if you wanted something else, holler.